notes/03-parameterized-components.md

```diff
@@ -1,6 +1,6 @@
 # Parameterized Components
```

Comment:
Not related to a11y, but this file has a typo, inconsistent capitalization, and inconsistent punctuation which I figured I can address.